### ISSUE TITLE
Serialization every 15 seconds

### ISSUE THREAD

**Issue by jhubers** [DynamoDS/Dynamo#6259]
## Dynamo version

0.9.1.4062
## Operating system

Windows 8.1
## What did you do?

Noticed while debugging that `protected override void SerializeCore(XmlElement element, SaveContext context)` runs every 15 seconds. Is that normal?
## What did you expect to see?

I expect that Serialization only runs when you save the *.dyn file

**Comment by ke-yu** [comment]
It should be related to auto backup. @Randy-Ma would have more idea about this part.

**Comment by Randy-Ma** [comment]
The auto backup runs every 60s by default. This can be changed to a different value and you can refer to the Dynamo FAQ in the wiki.

**Comment by ke-yu** [comment]
@jhubers FYI, you can find the backup frequency setting in the last question in [Dynamo FAQ](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-FAQ)

**Comment by jhubers** [comment]
@ke-yu Well, In fact `DynamoSettings.xm`l is here: `C:\Users\jhubers\AppData\Roaming\Dynamo\0.9` and changing `<BackupInterval>60000</BackupInterval>` has strange effects. If I set it to 120000 then Serialization runs sometimes every minute, sometimes after 15 seconds, sometimes after 40 sec etc. If I set it to 10000 there is no difference in this behaviour. If I set it to 6 then it runs all the time. Is there not a way to only run `protected override void SerializeCore(XmlElement element, SaveContext context)` when I save the file manually?
